pypdf/filters.py

```diff
@@ -41,12 +41,12 @@
 from io import BytesIO
 from typing import Any, Dict, List, Optional, Tuple, Union, cast
 
+from ._codecs._codecs import LzwCodec
```

Comment:
This indirectly exposes both the internal and the external implementation as a public module, including the encoder. Do we really want to do this? I tend to either make this import local or private by adding ` as _LzwCodec`.